src/main/steps/error/error.controller.ts

```diff
@@ -58,7 +58,7 @@ export class ErrorController {
     const language = req.session?.lang || SupportedLanguages.En;
     const errorText =
       errorContent[language][res.statusCode] || errorContent[language][StatusCodes.INTERNAL_SERVER_ERROR];
-    const commonContent = generatePageContent({
+    const commonContent = generateCommonContent({
```

Comment:
dont you also need to generate the page content here? not sure if it has one but just asking incase.